lib/design/app_theme.dart

```diff
@@ -6,6 +6,7 @@ class SeedsAppTheme {
   static ThemeData get lightTheme {
     return ThemeData(
         primaryColor: AppColors.primary,
+        backgroundColor: AppColors.primary,
```

Comment:
Why is background and primary color the same? Is there a reason for this - I mean in the future do we expect them to be different?

We can remove primary color really - what's primary mean....  

If we have two elements with the same color in there, there should be a very good reason